modules/config-fields-misc.adoc

```diff
@@ -54,4 +54,51 @@
 | xref:reference-miscellaneous-v2-ui[**FEATURE_UI_V2**] | Boolean | When set, allows users to try the beta UI environment. 
 
 *Default:* `True`
+
+| **FEATURE_REQUIRE_TEAM_INVITE**  | Boolean | Whether to require invitations when adding a user to a team + 
+  + 
+**Default:** True
+
+| **FEATURE_REQUIRE_ENCRYPTED_BASIC_AUTH**  | Boolean | Whether non-encrypted passwords (as opposed to encrypted tokens) can be used for basic auth + 
+  + 
+**Default:** False
+
+| **FEATURE_RATE_LIMITS**  | Boolean | Whether to enable rate limits on API and registry endpoints. Setting  FEATURE_RATE_LIMITS to `true` causes `nginx` to limit certain API calls to 30 per second. If that feature is not set, API calls are limited to 300 per second (effectively unlimited). + 
+  + 
+**Default:** False
+
+| **FEATURE_FIPS** | Boolean | If set to true, {productname} will run using FIPS-compliant hash functions + 
+ + 
+ **Default:** False
+| **FEATURE_ADVERTISE_V2** | Boolean | Whether the v2/ endpoint is visible + 
+ + 
+ **Default:** True
+| **FEATURE_AGGREGATED_LOG_COUNT_RETRIEVAL**  | Boolean |  Whether to allow retrieval of aggregated log counts + 
+ + 
+ **Default:** True
+ | **FEATURE_ANONYMOUS_ACCESS** | Boolean |  Whether to allow anonymous users to browse and pull public repositories + 
+  + 
+**Default:** True
+| **FEATURE_APP_REGISTRY** | Boolean |  Whether to enable support for App repositories + 
+  + 
+**Default:** False
```

Comment:
This is deprecated and codebase for Helm charts is removed. So this should also not be in the docs.Helm charts are now stored as OCI artifacts.